spec/defines/keepalived_vrrp_track_process_spec.rb

```diff
@@ -0,0 +1,104 @@
+require 'spec_helper'
+
+describe 'keepalived::vrrp::track_process', type: :define do
+  let(:facts) { debian_facts }
```

Comment:
Please use https://github.com/mcanevet/rspec-puppet-facts.

Reply:
This would be preferable.  Converting the rest of the module is on my todo list.